src/snowcli/cli/snowpark/package.py

```diff
@@ -37,10 +37,10 @@ def package_lookup(
     """
     Check to see if a package is available on the Snowflake anaconda channel.
     """
-    packageResponse = utils.parse_anaconda_packages([name])
-    ## if list has any items
+    package_response = utils.parse_anaconda_packages([name])
```

Comment:
Please separate task part and refactor. Create new PR for it

Reply:
I think the refactor is part of this task. Introducing precommit check without refactoring code will result in precommit raisning error for others (or main branch)